Aquality.Selenium/src/Aquality.Selenium/Browsers/BrowserFactory.cs

```diff
@@ -10,5 +13,21 @@ protected BrowserFactory()
         }
 
         public abstract Browser Browser { get; }
+
+        protected ArgumentOutOfRangeException LoggedWrongBrowserNameException
```

Comment:
Why do you need it here? For me it looks a little bit weird. If you want to do some common actions you can define template method in abstract class and log it in this method.

Reply:
okay, removed